gapic/src/main/com/google/gapid/perfetto/models/ThreadTrack.java

```diff
@@ -71,12 +74,12 @@
 
 
   private final ThreadInfo thread;
-  private final SliceTrack sliceTrack;
+  private final SliceFetcher sliceTrack;
```

Comment:
The variable name here and below is kept as before, is this intended because it's just a very thin wrapper?

Reply:
i didn't think it was worth renaming.